### ISSUE TITLE
add easily findable doc for users to know which versions of sdk & relayers each ibc-go version has been tested with

### ISSUE THREAD

**Issue by charleenfei** [cosmos/ibc-go#714]
## Steps to Reproduce

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged/assigned

**Comment by colin-axner** [comment]
What's the proposed action here? I assume the proposal is not that we go and test relayers to see if they work with our versions, but rather host a document which relayers can open a pr to show they have tested our versions?

**Comment by charleenfei** [comment]
yes, exactly. basically, make public the compatibility testing info we've been doing w the relayers

**Comment by gjermundgaraba** [comment]
Closing this for now. We might reopen this in the future, but this is not currently planned.
